core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -40,11 +41,13 @@ angular.module('oppia').directive('topNavigationBar', [
         'SidebarStatusService', 'LABEL_FOR_CLEARING_FOCUS', 'UserService',
         'SiteAnalyticsService', 'NavigationService', 'WindowDimensionsService',
         'DebouncerService', 'DeviceInfoService', 'LOGOUT_URL',
+        'SHOW_CLASSROOM_CALLOUT',
         function(
             $scope, $http, $window, $timeout, $translate,
             SidebarStatusService, LABEL_FOR_CLEARING_FOCUS, UserService,
             SiteAnalyticsService, NavigationService, WindowDimensionsService,
-            DebouncerService, DeviceInfoService, LOGOUT_URL) {
+            DebouncerService, DeviceInfoService, LOGOUT_URL,
+            SHOW_CLASSROOM_CALLOUT) {
```

Comment:
I don't see you defining this constant in this PR, is that because it already exists?

Reply:
Yes @brianrodri , it is working and the SHOW_CLASSROOM_CALLOUT flag exists in constants.ts, so I did not define it . I had also added 'require('constants.ts');'  command in top-navigation-bar.directive.ts.